pkg/cryptoauthlib/Makefile

```diff
@@ -0,0 +1,59 @@
+PKG_NAME=cryptoauthlib
+PKG_URL=https://github.com/MicrochipTech/cryptoauthlib
+PKG_VERSION=af8187776cd3f3faf8bed412eaf6ff7221862e19
+PKG_LICENSE=BSD-3-Clause
```

Comment:
Are we sure about this license here? According to the [file upstream](https://github.com/MicrochipTech/cryptoauthlib/blob/master/license.txt) depending on what is being used from the library different licenses apply.